scripts/toggle_logging.sh

```diff
@@ -4,15 +4,17 @@ CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 source "$CURRENT_DIR/variables.sh"
 source "$CURRENT_DIR/shared.sh"
+source "$CURRENT_DIR/capture_pane_helpers.sh"
```

Comment:
The helpers in this file should only be included in files that handle "capture pane" features. If `get_path` helper is now needed in more places then maybe:

- rename `capture_pane_helpers.sh` to be more generic
- or create a separate helpers file just for `get_path` function

Reply:
After looking at this more I think this would be clean:

- create a separate helpers file for `get_path`
- this new helpers file would be sourced in `scripts/toggle_logging.sh` and in `scripts/capture_pane_helpers.sh`

Reply:
Ok, I will think about it